package.json

```diff
@@ -70,6 +70,12 @@
         "command": "runme.resetEnv",
         "category": "Runme",
         "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.stopBackgroundTask",
+        "category": "Runme",
+        "title": "Dispose terminal session to stop background task."
+
```

Comment:
Let's add this to below section `commandPalette` to avoid having this command shown there.

Reply:
will do.